primitives/runtime-interface/test-wasm-deprecated/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
```

Comment:
While I understand why you create multiple wasm files, I think we should not do this. The problem I see is just the compilation time. We already spent a lot of time compiling and should not increase this more.

We can still do this test.

1. Move the methods into the `test-wasm` crate.
2. Test calling the new version by just calling the method.
3. Test calling the old method by defining the `ext_*` method manually and call this directly.

Reply:
I agree with reasoning, but what you proposing is far from perfect test, because it uses different abstraction than that it is aimed to test (abstraction would be "just put versions inside #[runtime_intrface] and everything just works).

I'll see what can I do about compilation times if it is much of a problem.

Reply:
This is solved.